bluesky/callbacks/core.py

```diff
@@ -347,20 +351,25 @@ class LiveRaster(CallbackBase):
     aspect : str or float, optional
        Passed through to `Axes.imshow`
 
+    ax : Axes, optional
+        matplotib Axes; if none specified, current axes (``plt.gca()``)
+        are used.
+
     See Also
     --------
     :class:`bluesky.callbacks.LiveMesh`.
     """
     def __init__(self, raster_shape, I, *,
                  clim=None, cmap='viridis',
-                 xlabel='x', ylabel='y', extent=None, aspect='equal'):
-        fig, ax = plt.subplots()
+                 xlabel='x', ylabel='y', extent=None, aspect='equal',
+                 ax=None):
+        if ax is None:
+            ax = plt.gca()
```

Comment:
This should still do `fix, ax = plt.subplots` if an axes is not passed in. This will re-use the last axes the use clicked on.